experimenter/experimenter/klaatu/client.py

```diff
@@ -0,0 +1,95 @@
+import io
+import json
+import zipfile
+
+import requests
+from django.conf import settings
+
+
+class KlaatuClient:
+    def __init__(self, workflow_name):
```

Comment:
Add type annotations to the methods, that'll help document the expected args and responses.